Help/FAQ.rst

```diff
@@ -55,6 +55,15 @@ There are also generic bears, which can be applied language independently on
 your code. Their capabilities and information can be seen
 `here <https://coala.io/#/languages>`__.
 
+What is a bear anyway?
+----------------------
+
+A bear is a set of rules that is used by coala packages to check for
```

Comment:
Is `packages` a right word for `coala`?

Reply:
Also, `do some analysis` is better than `to check`, IMO. A bear is meant to do some analysis on the source code. It can check your code for potential problems, calculate metrics and even provide corrections for your code. A bear contains the actual subroutine that is responsible for checking code for certain specifications.

Reply:
@ksdme @palash25 @sangamcs consider the definition(after suggested changes) stated below:
A bear is a routine that is used by a coala package to check your code for
lint issues. A group of these routines(bears) is used for defining certain
quality requirements in various languages. A bear can analyse code for potential problems, calculate metrics and even provide corrections for your code. 
While many of these bears for various languages are shipped along with coala,
you can easily write some of your own.

Reply:
@sangamcse would 'libraries' be a better replacement for 'packages' ?

Reply:
Why not just 'coala' like: `a bear is a set of rules used by coala` or `a bear is a routine used by coala to check ...`